### ISSUE TITLE
Ethanol

### ISSUE THREAD

**Issue by ryuben** [AlexTheGre4t/MCTNGIssues#7]
Ethanol only works in Fluid boiler boxes atm

not in combustion engines or biogas

**Comment by AlexTheGre4t** [comment]
I'll try updating Forestry, if that doesnt work, ill get in contact with mezz.

**Comment by Xariyl** [comment]
I'd recommend the latest "unstable" version. Ironically it is way more stable and a lot of bugs have been fixed, including the issues with the multi block farms themselves.

**Comment by AlexTheGre4t** [comment]
Yeah. I was planning to do that but it's just one of those staple mods that you just "forget" about :D

**Comment by ryuben** [comment]
Poke

**Comment by ryuben** [comment]
now works in an engine at 40rf/t
